### PR TITLE
feat: remove time-based upgrade

### PR DESCRIPTION
**Pull request by dudong2**
## Description

Time based upgrade feature is removed from [cosmos-sdk pr #8849](https://github.com/cosmos/cosmos-sdk/pull/8849) of cosmos-sdk. It reflects the work that was missing in the previous bump up process.
I'm not modified upgrade.proto in the pr, because the time variable in upgrade.proto is reverted on [cosmos-sdk pr #9534](https://github.com/cosmos/cosmos-sdk/pull/9534).

closes: #787 

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: x/upgrade/types/plan_test.go [hunk 1]
```diff
@@ -58,41 +57,12 @@ func TestPlanValid(t *testing.T) {
...
 				Name: "missing",
 				Info: "important",
 			},
 		},
-		"negative height": {
-			p: types.Plan{
-				Name:   "minus",
-				Height: -12345,
-			},
-		},
```
Comment (tkxkd0159) [inline x/upgrade/types/plan_test.go:95]: why do you remove test cases related to name, Height, IBC?

Comment (dudong2) [reply x/upgrade/types/plan_test.go]: I reverted it. thanks

File: x/upgrade/types/proposal_test.go [hunk 2]
```diff
@@ -26,9 +26,9 @@ func TestContentAccessors(t *testing.T) {
 	}{
 		"upgrade": {
 			p: types.NewSoftwareUpgradeProposal("Title", "desc", types.Plan{
-				Name: "due_time",
-				Info: "https://foo.bar",
-				Time: mustParseTime("2019-07-08T11:33:55Z"),
+				Name:   "due_time",
```
Comment (jaeseung-bae) [inline x/upgrade/types/proposal_test.go:29]: `Name: "due_time"` -> Are you intending `Name: "due_height"`?

**Review by jaeseung-bae — COMMENTED** [review]
This binary(`x/upgrade/types/__debug_bin`) is attached by accident. Is's it?

**Review by jaeseung-bae — APPROVED** [review]

**Review by zemyblue — APPROVED** [review]

**Pull request merged**